Packages/UIComponents/Sources/UIComponents/Models/AlertPresentationType.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  File.swift
+//  
+//
+//  Created by Eric Silverberg on 12/21/22.
+//
+
+import Foundation
+
+public enum DeepLinkDestination {
+    case none
+}
+
+public enum AlertPresentationModality {
+    case none
```

Comment:
I'd argue that `none` modality shouldn't exist, because errors that don't need to be displayed shouldn't be emitted at all, otherwise the UI would be deciding what to show, when it should be the view model.